File: modulo_2/kotlin/JetpackCompose/app/src/main/java/com/example/jetpackcompose/model/Pet.kt
package com.example.jetpackcompose.model
import androidx.annotation.DrawableRes

data class Pet(
    val id: Int,
    val name: String,
    val type: String,
    val age: Int,
    @DrawableRes val imageRes: Int,
    val description: String,
    val adopted: Boolean = false,
    val adoptedBy: String? = null,
    val createdAt: Long = System.currentTimeMillis(),
    val isFavorite: Boolean = false
)

File: modulo_2/flutter/pet_adoption_app_mp/lib/models/pet.dart
class Pet {
  final String name;
  final String type;
  final int age;
  final String image;
  final String description;
  bool isLiked;

  Pet({
    required this.name,
    required this.type,
    required this.age,
    required this.image,
    required this.description,
    this.isLiked = false,
  });
}
